tests/e2e/test_helper.go

```diff
@@ -42,36 +43,53 @@ var (
 	kubeCli kubernetes.Interface
 )
 
-type clusterFixture struct {
+type clusterSpec struct {
 	ns          string
 	clusterName string
-	cases       []testCase
+
+	// values override the specified helm values
+	values map[string]string
+}
+
+type clusterFixture struct {
+	clusterSpec
+
+	cases []testCase
 }
 
-type testCase func(ns, name string)
+type testCase func(cluster clusterSpec)
 
 var fixtures = []clusterFixture{
 	{
-		ns:          "ns-1",
-		clusterName: "cluster-name-1",
+		clusterSpec: clusterSpec{
+			ns:          "ns-1",
+			clusterName: "cluster-name-1",
+		},
 		cases: []testCase{
 			testCreate,
 			testScale,
 			testUpgrade,
 		},
 	},
 	{
-		ns:          "ns-1",
-		clusterName: "cluster-name-2",
+		clusterSpec: clusterSpec{
+			ns:          "ns-1",
+			clusterName: "cluster-name-2",
+			values: map[string]string{
+				"tidb.separateSlowLog": "true",
```

Comment:
Need a case with `"tidb.separateSlowLog": "false"`?

Reply:
This option is default to false, but in case of we change the defaults later, I will explicitly declare that case.